backend/src/main/nl/tudelft/ti2806/riverrush/game/GameTrack.java

```diff
@@ -22,32 +26,30 @@
 
     private static final long UPDATE_DELAY = 1000;
     public static final Integer TRACK_LENGTH = 100;
-    private static final Integer DISTANCE_INTERVAL = 5;
```

Comment:
I don't really know what you mean with an interval, as events are never repeated. We can set the y position on the track of every event as well as the x position.